nrf_802154/doc/CHANGELOG.rst

```diff
@@ -9,6 +9,35 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+nRF Connect SDK 1.7.0 - nRF 802.15.4 Radio Driver
+*************************************************
+
+Added
+=====
+
+* Adopted usage of the Zephyr temperature platform for the RSSI correction.
+* Support for the coexistence feature from :ref:`nrfxlib:mpsl`.
+* Support for nRF21540 FEM GPIO interface on nRF53 Series.
+
+Notable changes
+===============
+
+* Modified the 802.15.4 Radio Driver Transmit API.
+  It now allows specifying whether to encrypt or inject dynamic data into the outgoing frame, or do both.
+  The :c:type:`nrf_802154_transmitted_frame_props_t` type is used for this purpose.
+
+Bug fixes
+=========
+
+* Fixed an issue where it would not be possible to transmit frames with invalid Auxiliary Security Header if :kconfig:`CONFIG_NRF_802154_ENCRYPTION` was set to ``n``. (KRKNWK-11218)
+* Fix an issue with the IE Vendor OUI endianness. (KRKNWK-10633)
+* Fixed various bugs in the MAC Encryption layer. (KRKNWK-10646)
+
+Limitations
+===========
+
+* Application and device drivers (excluding those compliant with :ref:`nrfxlib:mpsl`) must not use IRQ priority higher than :c:macro:`NRF_802154_SWI_PRIORITY` and :c:macro:`NRF_802154_SL_RTC_IRQ_PRIORITY`.
+
```

Comment:
```suggestion
Known issues
============

* At the rare occasions during the use of BLE and 802.15.4 multiprotocol the device might assert. (KRKNWK-11384)
* Transmitting the 802.15.4 frame with the improperly populated Auxilliary Security Header might result in assert. (KRKNWK-11204)
```

Reply:
We don't track known issues outside of sdk-nrf. This cannot be added here. Add a comment to: https://github.com/nrfconnect/sdk-nrf/pull/5475

Reply:
@czeslawmakarski , please re-review.